configuration/configuration.go

```diff
@@ -6,22 +6,24 @@ import (
 	"strings"
 	"time"
 
+	commonconfig "github.com/fabric8-services/fabric8-common/configuration"
 	errs "github.com/pkg/errors"
 	"github.com/spf13/viper"
 )
 
 const (
-	defaultLogLevel         = "info"
-	varCleanTestDataEnabled = "clean.test.data"
-	varDBLogsEnabled        = "enable.db.logs"
-	varDeveloperModeEnabled = "developer.mode.enabled"
-	varDiagnoseHTTPAddress  = "diagnose.http.address"
-	varEnvironment          = "environment"
-	varHTTPAddress          = "http.address"
-	varLogJSON              = "log.json"
-	varLogLevel             = "log.level"
-	varMetricsHTTPAddress   = "metrics.http.address"
-	varSentryDSN            = "sentry.dsn"
+	defaultLogLevel                        = "info"
+	varCleanTestDataEnabled                = "clean.test.data"
+	varDBLogsEnabled                       = "enable.db.logs"
+	varCleanTestDataErrorReportingRequired = "clean.test.data.error.reporting.required"
```

Comment:
what is this for ? if you add a new option, you would have to add to the config.yaml too